src/frontend/screens/Settings/components/Gamescope.tsx

```diff
@@ -115,20 +116,36 @@ const Gamescope = () => {
 
   return (
     <div className="gamescopeSettings">
-      {/* Enable Upscale */}
+      {/* Enable Gamescope */}
       <div className="toggleRow">
         <ToggleSwitch
-          htmlId="gamescopeUpscaleToggle"
-          value={gamescope.enableUpscaling || false}
+          htmlId="gamescopeEnableToggle"
+          value={gamescope.enableGamescope || false}
           handleChange={() =>
             setGamescope({
               ...gamescope,
-              enableUpscaling: !gamescope.enableUpscaling
+              enableGamescope: !gamescope.enableGamescope
             })
           }
-          title={t('setting.gamescope.enableUpscaling', 'Enables Upscaling')}
+          title={t('setting.gamescope.enableGamescope', 'Enable Gamescope')}
         />
       </div>
+      {/* Enable Upscale */}
+      {gamescope.enableGamescope && (
+        <div className="toggleRow">
+          <ToggleSwitch
+            htmlId="gamescopeUpscaleToggle"
+            value={gamescope.enableUpscaling || false}
+            handleChange={() =>
+              setGamescope({
+                ...gamescope,
+                enableUpscaling: !gamescope.enableUpscaling
+              })
+            }
+            title={t('setting.gamescope.enableUpscaling', 'Enables Upscaling')}
+          />
+        </div>
+      )}
       {/* Upscale Settings */}
       {gamescope.enableUpscaling && (
```

Comment:
I wonder if the upscaling/limiter options should also be conditionally rendered with with gamescope enable check

because with the new code I think you can:
- enable gamescope
- enable upscaling
- disable gamescop
- upscaling checkbox is gone but upscaling options will still be visible